xml/hardening.xml

```diff
@@ -3445,6 +3445,327 @@ Account Expires: Never</screen>
+    <title>Restricting Graphical Session Logins</title>
+
+    <para>
+     To improve security on your server, avoid to use graphical user
+     environments at all. Graphical programs are often
+     not designed to be run as &rootuser; and are more likely to
+     contain security issues than console programs. If you require a graphical
+     login, use a non-&rootuser; login. Configure your system to disallow
+     &rootuser; login on graphical session.
+    </para>
+
+    <para>
+     To deny the &rootuser; user login to graphical sessions you
+     can apply the same basic steps as outlined in
+     <xref linkend="sec-sec-prot-restrict-root-tty"/>.
+     You only need to also add the <literal>pam_securetty</literal> module to
+     the PAM stack file belonging to the display manager, for example
+     <filename>/etc/pam.d/gdm</filename> for GDM. The graphical session also
+     runs on a TTY device, <literal>tty7</literal>, by default. Therefore,
+     when you restrict &rootuser; logins to
+     <literal>tty1</literal>, then a &rootuser; login in the
+     graphical session will be denied.
+    </para>
+   </sect2>
+
+   <sect2 xml:id="sec-sec-prot-restrict-root-ssh">
+    <title>Restricting SSH Logins</title>
+    <para>
+     By default the &rootuser; user is also allowed to login
+     into a machine remotely via the SSH network protocol (if the SSH port is
+     not blocked by the firewall). To restrict this possibility the following
+     change of the OpenSSH configuration needs to be performed:
+    </para>
+    <procedure>
+     <step>
+      <para>Edit <filename>/etc/ssh/sshd_config</filename> and adjust the following parameter:</para>
+      <screen>PermitRootLogin no</screen>
+     </step>
+     <step>
+      <para>Restart the <systemitem class="service">sshd</systemitem> service to make the changes effective:</para>
+      <screen>systemctl restart sshd.service</screen>
+     </step>
+    </procedure>
+    <note>
+     <para>
+      Using the PAM module <literal>pam_securetty</literal> is not suitable in
+      case of OpenSSH, because SSH logins not necessarily go through the PAM
+      stack during the auth phase (e.g. when using SSH public key authentication).
+      Also an attacker could be able to differentiate between a wrong password
+      and a successful login that was only rejected later on by policy.
+     </para>
+    </note>
+   </sect2>
+
+  </sect1>
+  <sect1 xml:id="sec-sec-prot-inactivity-logout">
+   <title>Setting an Inactivity Timeout for Interactive Shell Sessions</title>
+   <para>
+    It can be a good idea to terminate an interactive shell session after a
+    certain time of inactivity, for example to:
+   </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+      prevent open, unguarded sessions.
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       avoid waste of system resources.
+      </para>
+     </listitem>
+    </itemizedlist>
```

Comment:
I think an itemizedlist is not needed for just 2 alternatives, but it is not a serious problem.

Reply:
I've used inline text instead of the 2 items, comitted in 18deb0107.